### PR TITLE
Put artifacts under preview other page to the right place.

### PR DESCRIPTION
**Pull request by sima-zhu**
Make changes to this page: https://review.docs.microsoft.com/en-us/java/api/overview/azure/other?view=azure-java-preview&branch=smoke-test

### CODE REVIEW

File: docs-ref-mapping/reference-unified.yml [hunk 1]
```diff
@@ -651,6 +666,11 @@
     uid: azure.java.sdk.landingpage.services.monitor
     landingPageType: Service
     items:
+    - name: Opentelemetry Exporter
```
Comment (srnagar) [inline docs-ref-mapping/reference-unified.yml:669]:
```suggestion
    - name: OpenTelemetry Exporter
```

File: preview/docs-ref-mapping/reference-preview.yml [hunk 2]
```diff
@@ -225,6 +240,15 @@
       landingPageType: Service
       children:
       - com.microsoft.azure.management.mixedreality*
+  - name: Monitor
+    uid: azure.java.sdk.landingpage.services.monitor.pre
+    landingPageType: Service
+    items:
+    - name: Opentelemetry Exporter
```
Comment (srnagar) [inline preview/docs-ref-mapping/reference-preview.yml:247]:
```suggestion
    - name: OpenTelemetry Exporter
```

**Review by srnagar — APPROVED** [review]

**Review by samvaity — APPROVED** [review]
Thanks Sima!

File: docs-ref-mapping/reference-unified.yml [hunk 3]
```diff
@@ -323,6 +323,16 @@
       href: ~/docs-ref-services/core-http-netty-readme.md
       children:
       - com.azure.core.http.netty*
+    - name: Apache Avro Serializer
+      uid: azure.java.sdk.landingpage.services.sdk.core.ApacheAvroSerialzier
+      href: ~/docs-ref-services/core-serializer-avro-apache-readme.md
+      children:
+      - com.azure.core.serializer.avro.apache
+    - name: Tracing
```
Comment (scbedd) [inline docs-ref-mapping/reference-unified.yml:331]: This should go under `core` imo. It is for the other 3 languages.

File: package.json [hunk 4]
```diff
@@ -1124,14 +1124,6 @@
                 "inputPath": [],
                 "packageDownloadUrl": "https://repo1.maven.org/maven2"
             },
-            {
```
Comment (scbedd) [inline package.json:1127]: 👍

**Review by scbedd — APPROVED** [review]
Pending suggestions from @srnagar

File: docs-ref-mapping/reference-unified.yml (hunk 3 above)
Comment (sima-zhu) [reply docs-ref-mapping/reference-unified.yml:331]: Tracing is under core. It is parallel to serializer. They all belongs to core if you scroll up a bit.

**Pull request merged**